tests/unit/test-cmd/test.build.js

```diff
@@ -5,8 +5,8 @@ import {it, describe} from 'mocha';
 import {assert} from 'chai';
 import sinon from 'sinon';
 
-import build, {safeFileName, FileFilter} from '../../src/cmd/build';
-import {withTempDir} from '../../src/util/temp-dir';
+import build, {safeFileName, FileFilter} from '../../../src/cmd/build';
```

Comment:
mmm, these relative paths are getting a little bit nuts. I filed https://github.com/mozilla/web-ext/issues/470 to fix it later.